package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "devDependencies": {
+    "@babel/core": "^7.19.6",
+    "@babel/eslint-parser": "^7.19.1",
+    "@babel/plugin-syntax-jsx": "^7.18.6",
+    "@babel/preset-env": "^7.19.4",
+    "@babel/preset-react": "^7.18.6",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-react": "^7.31.10",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0"
+  }
+}
```

Comment:
Great job @Kelvin-Ben 🤝

- [ ] Kindly use `npx create-react-app my-app` command to create your react App. This command will generate a package.json file which looks something like the screenshot below.
![package](https://user-images.githubusercontent.com/63917725/197632510-8a27eca4-da8f-4c48-9bec-989635c9e7ea.PNG)

- [ ] When I checked your `package.json` file, I can see it is missing some scripts.
- [ ] You may also try to create your app again because when I start the app, I get the following error.
![error1](https://user-images.githubusercontent.com/63917725/197633002-6d1bbfdc-7ad4-494b-a3fa-8f7000cc6f00.PNG)